apps/saved_searches/__init__.py

```diff
@@ -119,6 +120,22 @@ def report():
                     user_data['last_report'] = now
                     user_data['next_report'] = get_next_date(scheduling)
                     do_update = True
+
+            subscribed_desks = search['subscribers'].get('desk_subscriptions', [])
+            for desk_data in subscribed_desks:
```

Comment:
seems like there is a lot of duplicated code in this and previous `for` cycle, would it be possible to avoid that?